sdks/java/testing/test-utils/build.gradle

```diff
@@ -30,12 +31,15 @@ dependencies {
   compile project(path: ":sdks:java:core", configuration: "shadow")
   compile library.java.vendored_guava_26_0_jre
   compile library.java.google_cloud_bigquery
-  compile project(":sdks:java:extensions:google-cloud-platform-core")
+  compile "com.google.code.gson:gson:2.8.6"
+  compile library.java.joda_time
+  compile "org.apache.commons:commons-compress:1.20"
+  compile "org.apache.commons:commons-lang3:3.9"
+  compile "org.apache.httpcomponents:httpclient:4.5.10"
```

Comment:
These don't currently have a common definition in BeamModulePlugin, but they should. BEAM-11669